helm-chart/values.yaml

```diff
@@ -137,3 +137,20 @@ test:
   enabled: false
   schedule: '@daily'
   baseUrl: "http://127.0.0.1:7546"
+
+livenessProbe:
```

Comment:
nit: Prefer them sort inline like the rest of the file instead of at the end

Reply:
Done

Reply:
I meant the top level key within the entire file, but that's fine 😄

Reply:
Next update to the chart

Reply:
LGTM